File: main/src/main/java/net/fwitz/math/main/binary/split/functions/trig/SplitSinZ.java
package net.fwitz.math.main.binary.split.functions.trig;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitSinZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("sin(z)")
                .computeFn(SplitComplex::sin)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/trig/SplitSinZ.kt
package net.fwitz.math.main.binary.split.functions.trig

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitSinZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("sin(z)")
        .computeFn { z -> z.sin }
        .render()
}